src/libasr/ASR.asdl

```diff
@@ -217,7 +217,6 @@ expr
         -- Such as: (x, y+z), (3.0, 2.0) generally not known at compile time
     | ComplexConstructor(expr re, expr im, ttype type, expr? value)
     | NamedExpr(expr target, expr value, ttype type)
-    | Compare(expr left, cmpop op, expr right, ttype type, expr? value, expr? overloaded)
```

Comment:
Also you need to put in `OverloadedCompare` node to represent the `expr? overloaded`.